code/__DEFINES/directional.dm

```diff
@@ -28,14 +28,98 @@
 /// Inverse direction, taking into account UP|DOWN if necessary.
 #define REVERSE_DIR(dir) ( ((dir & 85) << 1) | ((dir & 170) >> 1) )
 
+// Wallening todo: temporary helper, until we finish fleshing things out and can convert the main one
+// Why are we not just changing the sprites agian?
+#define INVERT_MAPPING_DIRECTIONAL_HELPERS(path, offset) \
+##path/directional/north {\
+	dir = SOUTH; \
+	MAP_SWITCH(pixel_z, pixel_y) = offset; \
+} \
+##path/directional/south {\
+	dir = NORTH; \
+	MAP_SWITCH(pixel_z, pixel_y) = -offset; \
+} \
+##path/directional/east {\
+	dir = WEST; \
+	pixel_x = offset; \
+} \
+##path/directional/west {\
+	dir = EAST; \
+	pixel_x = -offset; \
+}
+
+/// Directional helpers for things that use the wall_mount element
+#define WALL_MOUNT_DIRECTIONAL_HELPERS(path) \
+##path/directional/north {\
+	dir = SOUTH; \
+	MAP_SWITCH(pixel_z, pixel_y) = 35; \
+} \
+##path/directional/south {\
+	dir = NORTH; \
+	MAP_SWITCH(pixel_z, pixel_y) = -8; \
+} \
+##path/directional/east {\
+	dir = WEST; \
+	pixel_x = 11; \
+	MAP_SWITCH(pixel_z, pixel_y) = 16; \
+} \
+##path/directional/west {\
+	dir = EAST; \
+	pixel_x = -11; \
+	MAP_SWITCH(pixel_z, pixel_y) = 16; \
+}
```

Comment:
they combo with how sprites are laid out and are (by and large) designed to shift the sprite "onto" a wall.
Yes this can be defined, it is used elsewhere for setDir hooking